plugins/dell-k2/fu-dell-k2-ec.c

```diff
@@ -114,6 +119,12 @@ fu_dell_k2_ec_is_dev_present(FuDevice *device, guint8 dev_type, guint8 sub_type,
 	return fu_dell_k2_ec_dev_entry(device, dev_type, sub_type, instance) != NULL;
 }
 
+gboolean
+fu_dell_k2_ec_is_uod_supported(FuDevice *device)
```

Comment:
I don't think the helper is useful, maybe just use `fu_device_has_private_flag` directly?

Reply:
Yeah I think so too; it's only used in one place.

Reply:
fixed.